### ISSUE TITLE
Job Candidates View - v1.0

### ISSUE THREAD

**Issue by wdprice** [topcoder-platform/taas-app#10]
![image](https://user-images.githubusercontent.com/6014049/100250880-2746da80-2f0c-11eb-921c-835ff6d06fe8.png)

Lists all jobCandidates associated to a particular job. 

Skills list & match % comes from jobSkills and userSkills respectively.

Resume link should download the resume directly from RecruitCRM.

Interested in Candidate assigns them to a shortlist, and the appear on the Interested tab. Not interested disqualified the candidate.

All interactions should sync to RecruitCRM (see RecruitCRM Integration issues for details)

"Schedule Interview" button should be hidden and considered out of scope at this time.

**Comment by SathyaJayabal** [comment]
@maxceem , the `Download Resume` link will be available only after RCRM integration, Am I right?

**Comment by sandhiyakavi** [comment]
The Skill match % calculation is incorrect ,already raised issue https://github.com/topcoder-platform/taas-app/issues/33

Others are working fine.


https://user-images.githubusercontent.com/55895538/102860842-f063d600-4471-11eb-9ade-404caa2ff492.mp4

**Comment by maxceem** [comment]
@SathyaJayabal can we treat this as passed review on DEV? as we are tracking skills matching issue separately?

**Comment by SathyaJayabal** [comment]
@maxceem , how about the download resume link implementation?  This is still pending.

**Comment by maxceem** [comment]
I remember Will asked us to remove Resume link for now. @wdprice, could you please confirm, that we remove the resume link for now?

**Comment by wdprice** [comment]
I don't recall requesting that. I'd ideally like to have the link displayed, and have it get the Resume attached to the RCRM profile.

@nkumar-topcoder @maxceem - when adding jobCandidate externalId, lets also add a field to track the resume link. We can try to populate that w/ info during the Zap integration.

**Comment by wdprice** [comment]
@nkumar-topcoder @maxceem 

The resume URL is returned in the RCRM API: 
![image](https://user-images.githubusercontent.com/6014049/103905164-3fde0f00-50cc-11eb-8614-89f867c984d4.png)

I also confirmed it is included in the jobCandidate Zapier integration.

**Comment by maxceem** [comment]
@SathyaJayabal resume link is now shown for Job Candidates. See example Job http://localhost:8080/taas/myteams/16870/positions/9eaa6f3f-36f9-4e99-9902-274bfa6f5db8.

**Comment by SathyaJayabal** [comment]
Verified on dev

**Comment by SathyaJayabal** [comment]
verified in prod.
Candidate status update issue is tracked in https://github.com/topcoder-platform/taas-app/issues/25
<img width="1440" alt="Screenshot 2021-01-28 at 8 00 01 PM" src="https://user-images.githubusercontent.com/58783823/106152424-6eb74480-61a3-11eb-9355-a895ec1d89af.png">
